include/swift/IDE/Formatting.h

```diff
@@ -20,8 +20,8 @@ struct CodeFormatOptions {
 public:
   bool UseTabs = false;
   bool IndentSwitchCase = false;
-  unsigned IndentWidth = 4;
-  unsigned TabWidth = 4;
+  unsigned IndentWidth = 2;
+  unsigned TabWidth = 8;
```

Comment:
I wasn't sure if that would be a problem for Xcode, so I figured better to just discuss it :-).  What do you think of changing the defaults in `swift-format`?